pipelines/sqre/infra/build_salsciplatlab.groovy

```diff
@@ -0,0 +1,89 @@
+node('jenkins-master') {
+  dir('jenkins-dm-jobs') {
+    checkout([
+      $class: 'GitSCM',
+      branches: scm.getBranches(),
+      userRemoteConfigs: scm.getUserRemoteConfigs(),
+      changelog: false,
+      poll: false
+    ])
+    notify = load 'pipelines/lib/notify.groovy'
+    util = load 'pipelines/lib/util.groovy'
+  }
+}
+
+notify.wrap {
+  util.requireParams([
+    'BASE_IMAGE',
+    'IMAGE_NAME',
+    'JLBLEED',
+    'NO_PUSH',
+    'TAG',
+    'TIMEOUT',
+  ])
+
+  String tag         = params.TAG
+  Boolean pushDocker = (! params.NO_PUSH.toBoolean())
+  Boolean jlBleed    = params.JLBLEED
+  String pyver       = params.PYVER // use as opaque string
+  String baseImage   = params.BASE_IMAGE
+  String imageName   = params.IMAGE_NAME
+  Integer timelimit  = params.TIMEOUT
+
+  def run = {
+    stage('checkout') {
+      def branch = 'prod'
+      if (jlBleed) {
+        branch = 'master'
+      }
+      git([
+        url: 'https://github.com/lsst-sqre/sal-sciplat-lab',
```

Comment:
Is this job identical to the original sciplatlab accept for the source gitr repo?